package.json

```diff
@@ -45,13 +45,18 @@
     "@babel/core": "^7.8.0",
     "@babel/preset-env": "^7.8.3",
     "@babel/preset-react": "^7.7.4",
+    "@fluent/bundle": "^0.15.0",
+    "@fluent/langneg": "^0.3.0",
+    "@fluent/react": "^0.11.1",
     "@sentry/browser": "^5.10.2",
     "autoprefixer": "^9.7.3",
     "babel-core": "^6.26.3",
     "babel-loader": "^8.0.6",
     "classnames": "2.2.6",
+    "core-js": "^3.6.4",
```

Comment:
Do we know why this is a direct dependency?

Reply:
[@babel/polyfill has been deprecated](https://babeljs.io/docs/en/babel-polyfill) in favor of `core-js` + `regenerator-runtime`.

If I remove them, I get a `regeneratorRuntime is not defined` error. Looking closely at the polyfill documentation, it may only be needed for the generator function in l10n.js? Let’s see it replacing it with a simple array allows us to drop these two dependencies

Reply:
Confirmed, let’s remove them!